Lib9c/Action/Garages/UnloadFromGarages.cs

```diff
@@ -0,0 +1,143 @@
+#nullable enable
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Security.Cryptography;
+using Bencodex.Types;
+using Lib9c.Abstractions;
+using Libplanet.Action;
+using Libplanet.Action.State;
+using Libplanet.Common;
+using Libplanet.Crypto;
+using Libplanet.Types.Assets;
+using Nekoyume.Model.State;
+
+namespace Nekoyume.Action.Garages
+{
+    [ActionType("unload_from_garages")]
+    public class UnloadFromGarages : GameAction, IUnloadFromGaragesV1, IAction
+    {
+        public IReadOnlyList<(
+                Address recipientAvatarAddress,
+                IOrderedEnumerable<(Address balanceAddress, FungibleAssetValue value)>?
+                fungibleAssetValues,
+                IOrderedEnumerable<(HashDigest<SHA256> fungibleId, int count)>? fungibleIdAndCounts,
+                string? memo)>
+            UnloadData { get; private set; }
+
+        public UnloadFromGarages()
+        {
+            UnloadData = new List<(
+                Address recipientAvatarAddress,
+                IOrderedEnumerable<(Address balanceAddress, FungibleAssetValue value)>?
+                fungibleAssetValues,
+                IOrderedEnumerable<(HashDigest<SHA256> fungibleId, int count)>?
+                fungibleIdAndCounts, string? memo)>();
+        }
+
+        public UnloadFromGarages(
+            IReadOnlyList<(
+                Address recipientAvatarAddress,
+                IEnumerable<(Address balanceAddress, FungibleAssetValue value)>?
+                fungibleAssetValues,
+                IEnumerable<(HashDigest<SHA256> fungibleId, int count)>? fungibleIdAndCounts,
+                string? memo)> unloadData)
```

Comment:
It is too sad to duplicate codes for type because of csharplang's limitation. 😢

Reply:
```
type Foo = (...)
```
😢